cmd/job-executor-service-initcontainer/main.go

```diff
@@ -51,6 +54,8 @@ type envConfig struct {
 	OAuthDiscovery string `envconfig:"OAUTH_DISCOVERY" required:"false"`
 	// The gitCommitId of the initial cloud event, for older Keptn instances this might be empty
 	GitCommitID string `envconfig:"GIT_COMMIT_ID"`
+	// The URL to the resource service
+	ConfigurationServiceURL string `envconfig:"CONFIGURATION_SERVICE" default:"resource-service:8080"`
```

Comment:
This default value will never work, as it's usually not running in the same namespace as `keptn`.
I recommend setting required true.

Reply:
Actually, is this needed? Doesn't look like `ConfigurationServiceURL` is used in main.go

Reply:
No, this is not needed in the `init-container` and has been removed.